sdk/src/events.ts

```diff
@@ -49,6 +49,10 @@ export const CALLBACK_HANDLER_EVENTS = {
   CallbackReceived: 'CallbackReceived',
 };
 
+export const PROXY_FACTORY_EVENTS = {
```

Comment:
couldn't we remove these custom definitions with the task that exports the missing events from the contracts?

Reply:
How would you do it? `TypedEvent`s don't have their name as a field, e.g.,
```ts
export type PluginRepoRegisteredEvent = TypedEvent<
  [string, string] & { subdomain: string; pluginRepo: string }
>;
// ...
export interface TypedEvent<EventArgs extends Result> extends Event {
  args: EventArgs;
}
```

Reply:
If we move away from hardhat-ethers to hardhat-viem. We don't need this typechain stuff anymore, luckily.

Reply:
the event names are in the interface with the whole selector
![image](https://github.com/aragon/osx-commons/assets/4057473/7b86ed85-13eb-48d7-a4c3-eceacfbe18cc)

but I meant when we need to type every event anyway it would hurt to provide this with the ethers package. typing it twice isn't that much more work then typing it once